code_challenges/advent_of_code/program_alarm.py

```diff
@@ -0,0 +1,55 @@
+def opcode_integers():
+    with open("./code_challenges/advent_of_code/opcode_input") as integers:
+        opcode = integers.read().split(",")
+
+    return [int(integer) for integer in opcode]
+
+
+def get_next_noun_verb_pair():
+    for i in range(100):
+        for j in range(100):
+            noun = i
+            verb = j
+            pair = (noun, verb)
+            yield pair
+
```

Comment:
This should raise a StopIteration outside the for loops to indicate that it is done.

Reply:
you are correct i must have accidentally removed it